### PR TITLE
Change default model settings

### PR DESCRIPTION
**Pull request by darylchang123**
Made the following changes:
- No dropout in conv layers
- Change dropout in FC to 0.7
- Use batch norm by default
- Use batch norm before ReLU (to do so, I moved ReLU into a separate layer)
- Change learning rate to 0.01

Tested using ml_utils.build_model() to make sure it runs correctly.

### CODE REVIEW

**Review by technoapurva — APPROVED** [review]
LGTM
